genie_registry/clinical.py

```diff
@@ -174,7 +174,27 @@ def _check_int_year_consistency(
 
     return ""
 
+def _check_year_death_validity(clinicaldf: pd.DataFrame) -> str:
+    """
+    Check if YEAR_DEATH >= YEAR_CONTACT.
+    When YEAR_DEATH is known, YEAR_CONTACT should = YEAR_DEATH.
 
+    Args:
+        clinicaldf: Clinical Data Frame
+        
+    Returns:
+        Error message if YEAR_DEATH if invalid
+    """    
+    error = ""
+    try:
+        check_result = np.where((pd.isna(clinicaldf['YEAR_DEATH']) | pd.isna(clinicaldf['YEAR_CONTACT'])), 'N/A', clinicaldf['YEAR_DEATH'] >= clinicaldf['YEAR_CONTACT'])
+        assert not "False" in check_result
+    except Exception:
+        error = (
+            f"Please double check the YEAR_DEATH column. "
```

Comment:
Could you change the validation message to something like:
`"Patient Clinical File: Please double check your YEAR_DEATH and YEAR_CONTACT columns. YEAR_DEATH must be >= YEAR_CONTACT."`

Reply:
Sure.